src/payment/checkout/payment-form/flex-microform/FlexMicroform.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { setPaymentTokenFetch } from '../../../payment-methods/cybersource/service';
+
+class FlexMicroform extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            microform: null,
+        }
+        this.onLoad = this.onLoad.bind(this);
+        this.getPaymentToken = this.getPaymentToken.bind(this);
+    }
+
+    componentDidMount() {
+        setPaymentTokenFetch(this.getPaymentToken);
+
+        const script = document.createElement('script');
+        script.src = "https://flex.cybersource.com/cybersource/assets/microform/0.11/flex-microform.min.js";
```

Comment:
Performance implications of this?

Reply:
I'd worry more about whether their script is idempotent.  It probably is, but if it's not, this could create some really crazy/broken behavior if it gets called multiple times (see above)

But yeah, we'll have to wait for their script before showing the UI, presumably... It's only 7.3k though.